packages/sui-studio-utils/src/domain-builder/index.js

```diff
@@ -10,6 +10,13 @@ export default class DomainBuilder {
     this._useCases = {}
   }
 
+  mockUseCases(mocks) {
+    mocks.forEach(
```

Comment:
You may want to re-use the existing methods to exploit the validation logic they already implement, like the respondWith method validation for success and fail:
```
mocks.forEach(([useCase, responses]) =>
  this.for({useCase}).respondWith(responses)
)
```